doc/faq.rst

```diff
@@ -123,3 +123,40 @@ If you installed via ``pip``
 and the output description is incorrect or ``None``,
 you have another module installed that is also named ``eos``.
 Mind that the package name for the ``pip3 install`` command is ``eoshep``.
+
+.. _faq-tags:
+
+... choose which ``tag`` to use for :math:`b \to s \ell \ell` observables
+=========================================================================
+
+The contributions from non-local operators in :math:`b \to s \ell \ell` observables
+can be estimated with different theoretical approaches, usually not valid in the same energy range,
+based on different input parameters and different theoretical assumptions.
+Depending on the analysis under consideration, the user can choose between these approaches by specifying an option called ``tag``.
+As different ``tags`` hide completely different assumptions, we have made this option mandatory.
+
+We collect here the different ``tags`` used in EOS and the corresponding references.
+
+.. list-table::
+   :widths: 15 55 30
+   :header-rows: 1
+
+   * - ``tag``
+     - Description
+     - Reference
+
+   * - ``Naive``
+     - Leading-order calculation.
```

Comment:
Hm, I disagree. ``Absence of any non-local contributions.``?

Reply:
Not really, I think we are still using `C9eff` and `C7eff` at leading order in alpha_s, see e.g. https://github.com/eos/eos/blob/f3c073bf2bb9a2aa2af7740ab4a905a68c2c90b8/eos/rare-b-decays/lambda-b-to-lambda1520-ll-naive.cc#L88-L89

Reply:
The key is in the name of the class: ``c7eff`` and ``c9eff`` are the short-distance quantities, basically ``c7`` and ``c9`` modified to be invariant under the RGE to higher order in $\alpha_s$. As such, they are essential to complete the  local contributions but do not contain any non-local terms (despite arising from four-quark operators).

Reply:
Ah yes sorry I am an idiot